tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+"""
+Handshake tests against openssl using TLS13.
+At the moment these tests are expected fail, as TLS13 is incomplete.
+"""
+
+import argparse
+import os
+import sys
+
+from common.s2n_test_openssl import run_openssl_connection_test
+from common.s2n_test_scenario import get_scenarios, Mode, Cipher, Version
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Runs TLS1.3 minimal handshake integration tests against Openssl')
```

Comment:
Maybe add what version of OpenSSL the integration test is running against as a part of the description.

Reply:
The test prints out that info when it runs-- see line 40. I'd rather that the description / help messaging be a simple string, without any code execution.